src/lfortran/semantics/ast_common_visitor.h

```diff
@@ -359,7 +359,7 @@ class CommonVisitorMethods {
         diag.semantic_warning_label(
             ".xor. is an LFortran extension",
             {x.base.base.loc},
-            "please help us: what is the Fortran way to specify xor?"
+            ""
```

Comment:
Maybe put this in, that way there is some text next to the line.
```suggestion
            "LFortran extension"
```
My eyes usually look at the line first, so if the short text is self explanatory, then that's all I need to know.

Reply:
Okay sure!